onnxruntime/core/mlas/lib/amd64/ErfKernelFma3.asm

```diff
@@ -0,0 +1,567 @@
+        vbroadcastss ymm14,ErfConstants.Exp_P1[rax]
+
+        vbroadcastss ymm13,ErfConstants.Exp_P2[rax]
+        vfmadd213ps  ymm8,ymm0,ymm14        ; *+ exp_p1
+        vfmadd213ps  ymm9,ymm1,ymm14
+        vfmadd213ps  ymm10,ymm2,ymm14
+        vfmadd213ps  ymm11,ymm3,ymm14
+
+        vbroadcastss ymm12,ErfConstants.Exp_P3[rax]
+        vfmadd213ps  ymm8,ymm0,ymm13        ; *+ exp_p2
+        vfmadd213ps  ymm9,ymm1,ymm13
+        vfmadd213ps  ymm10,ymm2,ymm13
+        vfmadd213ps  ymm11,ymm3,ymm13
+
+        vbroadcastss ymm15,ErfConstants.Exp_P4[rax]
+        vfmadd213ps  ymm8,ymm0,ymm12        ; *+ exp_p3
+        vfmadd213ps  ymm9,ymm1,ymm12
+        vfmadd213ps  ymm10,ymm2,ymm12
+        vfmadd213ps  ymm11,ymm3,ymm12
+
+        vbroadcastss ymm14,ErfConstants.Exp_P5[rax]
+        vfmadd213ps  ymm8,ymm0,ymm15        ; *+ exp_p4
+        vfmadd213ps  ymm9,ymm1,ymm15
+        vfmadd213ps  ymm10,ymm2,ymm15
+        vfmadd213ps  ymm11,ymm3,ymm15
+
+        vbroadcastss ymm13,ErfConstants.Exp_P6[rax]
+        vfmadd213ps  ymm8,ymm0,ymm14        ; *+ exp_p5
+        vfmadd213ps  ymm9,ymm1,ymm14
+        vfmadd213ps  ymm10,ymm2,ymm14
+        vfmadd213ps  ymm11,ymm3,ymm14
+
+        vbroadcastss ymm12,ErfConstants.Exp_X7F[rax]
+        vfmadd213ps  ymm8,ymm0,ymm13        ; *+ exp_p6
+        vfmadd213ps  ymm9,ymm1,ymm13
+        vfmadd213ps  ymm10,ymm2,ymm13
+        vfmadd213ps  ymm11,ymm3,ymm13
+
+        vcvttps2dq   ymm4,ymm4
+        vcvttps2dq   ymm5,ymm5
+        vcvttps2dq   ymm6,ymm6
+        vcvttps2dq   ymm7,ymm7
+
+
+        vbroadcastss ymm15,ErfConstants.ErfOne[rax]
+        vpaddd       ymm4,ymm4,ymm12        ; +127
+        vpaddd       ymm5,ymm5,ymm12
+        vpaddd       ymm6,ymm6,ymm12
+        vpaddd       ymm7,ymm7,ymm12
+
+        vpslld       ymm4,ymm4,23
+        vpslld       ymm5,ymm5,23
+        vpslld       ymm6,ymm6,23
+        vpslld       ymm7,ymm7,23
+
+        vmulps       ymm8,ymm8,ymm4         ; 2^i **exp(vf)
+        vmulps       ymm9,ymm9,ymm5
+        vmulps       ymm10,ymm10,ymm6
+        vmulps       ymm11,ymm11,ymm7
+
+        vsubps       ymm8,ymm15,ymm8
+        vsubps       ymm9,ymm15,ymm9
+        vsubps       ymm10,ymm15,ymm10
+        vsubps       ymm11,ymm15,ymm11
+
+        ; merge small numbers' result
+        vorps        ymm8,ymm8,YMMWORD PTR ErfKernelFrame.ErfBuffer1[rsp]
+        vorps        ymm9,ymm9,YMMWORD PTR 32+ErfKernelFrame.ErfBuffer1[rsp]
+        vorps        ymm10,ymm10,YMMWORD PTR 64+ErfKernelFrame.ErfBuffer1[rsp]
+        vorps        ymm11,ymm11,YMMWORD PTR 96+ErfKernelFrame.ErfBuffer1[rsp]
+
+        ; copy sign
+        vorps        ymm0,ymm8,YMMWORD PTR ErfKernelFrame.ErfBuffer0[rsp]
+        vorps        ymm1,ymm9,YMMWORD PTR 32+ErfKernelFrame.ErfBuffer0[rsp]
+        vorps        ymm2,ymm10,YMMWORD PTR 64+ErfKernelFrame.ErfBuffer0[rsp]
+        vorps        ymm3,ymm11,YMMWORD PTR 96+ErfKernelFrame.ErfBuffer0[rsp]
+
+        vmovups YMMWORD PTR [rdx],ymm0
```

Comment:
same comments about using rdx+32 instead of 32[rdx]

Reply:
All fixed.